src/assets/scss/style.scss

```diff
@@ -1,7 +1,67 @@
 body {
   // background: #f5f5f5;
 }
+<<<<<<< HEAD
```

Comment:
Could you perfect the rebase and remove this from your code ouput

Reply:
resolved